signing/src/test/java/tech/pegasys/web3signer/signing/config/metadata/parser/YamlSignerParserTest.java

```diff
@@ -233,7 +235,7 @@ void keyStoreMetaDataInfoWithHttpURIFailsWithException() throws IOException {
 
   @Test
   void aSignerIsCreatedForEachMatchingFactory() throws IOException {
-    lenient().when(otherBlsArtifactSignerFactory.getKeyType()).thenReturn(KeyType.BLS);
+    lenient().when(secpArtifactSignerFactory.getKeyType()).thenReturn(KeyType.BLS);
```

Comment:
Should return a KeyType of secp in this case

Reply:
Originally it wasn't, but I've fixed it to use secp type correctly.